herbert_core/classes/data_loaders/@spe/private/put_spe.m

```diff
@@ -26,18 +26,16 @@
 filename=[name,ext];
 filepath=[path,filesep];
 
-% Prepare data for Fortran routine: must get rid of NaNs
+% Remove NaNs from data
```

Comment:
was this block of code *only* required for FORTRAN? in which case the whole section can be dropped rather than the comment changing

Reply:
in SPE it still need to write as -1E+30 or whatever, for compartibility with old crap may laying around, so the replacing NaNs block is necessary.